db_processor.bal

```diff
@@ -404,6 +414,18 @@ function checkMedicalNeedQuantityAvailable(AidPackageItem aidPackageItem) return
 
 }
 
+//Check Already Pledged against the Aid Package Item Update
+function checkAlreadyPledgedAgainstAidPackageUpdate(AidPackageItem aidPackageItem, boolean deleteItem) returns boolean|error {
+    decimal totalAmount = check dbClient->queryRow(`SELECT sum(TOTALAMOUNT) FROM AID_PACKAGE_ITEM WHERE PACKAGEID=${aidPackageItem.packageID} AND PACKAGEITEMID!=${aidPackageItem.packageItemID};`);
+    Quotation quotation = check getQuotation(aidPackageItem.quotationID);
```

Comment:
Isn't this required only if `deleteItem` is false? How about moving it into the if block?

Reply:
no this is required at the aidpackage items quantity modification and the delete both.